meilisearch-lib/src/index/index.rs

```diff
@@ -175,12 +175,10 @@ impl Index {
             two_typos: Setting::Set(self.min_word_len_two_typos(txn)?),
         };
 
-        let disabled_words = self
-            .exact_words(txn)?
-            .into_stream()
-            .into_strs()?
-            .into_iter()
-            .collect();
+        let disabled_words = match self.exact_words(txn)? {
+            Some(fst) => fst.into_stream().into_strs()?.into_iter().collect(),
+            None => Default::default(),
+        };
```

Comment:
We are sure that it is a `Vec` here, right? And that it doesn't cost anything to construct a default one. Can we change this line a little bit? Just to be sure, for the future.

```suggestion
        let disabled_words = match self.exact_words(txn)? {
            Some(fst) => fst.into_stream().into_strs()?.into_iter().collect(),
            None => Vec::new(),
        };
```